mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, view: SpectrumViewerWindowView, main_window: MainWindowView):
         self.model = SpectrumViewerWindowModel(self)
         self.export_mode = ExportMode.ROI_MODE
         self.main_window.stack_changed.connect(self.handle_stack_modified)
+        self.selected_export_format = "CSV"
+        self.selected_export_area = "All"
```

Comment:
Probably better to get the values from the drop downs when needed than having to keep them synced in the presenter.